tools/ark-plugins/README.md

```diff
@@ -13,3 +13,45 @@ Run a Docker image build from the [Dockerfile](Dockerfile) as an init container
 ```bash
 ark plugin add {yourRepo/imageName:tag}
 ```
+
+## Add a plugin
```

Comment:
`Add a plugin` -> `Create a new plugin` ?

Reply:
I changed it to `Create a new plugin`